MNTOR-4151: Use higher-res images for the logo in emails

Vinnl opened the pull request:
<!-- The following is intended to be helpful to you. Feel free to remove anything that is not. -->

# References:

Jira: MNTOR-4151
Figma:

<!-- When adding a new feature: -->

# Description

This might make the logo look sharper on hi-res screens.

# Screenshot (if applicable)

There's no change for the regular header, but apparently the logo was a bit further away in the hero header with the purple background, and I think the font is a bit different.

Old:

![image](https://github.com/user-attachments/assets/d9a45a27-b462-464b-91e7-9d8d92e8d35e)

New:

![image](https://github.com/user-attachments/assets/82884047-9551-4eb0-9507-74050e53f3b4)

# How to test

Use the admin tool to send both the free and Plus monthly activity emails to a Mailinator address. The image URLs will then be served from localhost, so you can run the server to see what they look like in Mailinator.

# Checklist (Definition of Done)

- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, visual
- [x] If this PR implements a feature flag or experimentation, I've checked that it still works with the flag both on, and with the flag off.
- [x] If this PR implements a feature flag or experimentation, the Ship Behind Feature Flag status in Jira has been set
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

flozia reviewed (approved):
Right now the logo with the background has an additional padding of `10px`. This matches the previous images, but wouldn’t the images ideally have the same size?

Vinnl commented:
@flozia Yeah, we added the burnt-in padding when we added the background, to make it slightly less jarring in dark mode, but that does result in some extra padding for that image. We should probably add a negative margin to that image in the regular header, but... Email is scary, it's not a huge issue, and apparently we've been living with it for a while, so... It's probably the best use of our time to leave it like this.
